pandas/tests/groupby/test_functional.py

```diff
@@ -0,0 +1,371 @@
+# -*- coding: utf-8 -*-
+
+""" test function application """
```

Comment:
If I hadn't looked at the filename, I would have been completely confused by this module docstring.